frontend/src/app/modules/workspace/personal/personal.component.ts

```diff
@@ -44,58 +22,153 @@ const ELEMENT_DATA = [
-  //now i can`t get current user and his team cause not implementer this services
-  //it's mock team and user id
-  team: number = 1;
-  user: number = 4;
-  currentFolder: number | undefined;
 
   displayedColumns: string[] = ['name', 'owner', 'details'];
-  dataSource = ELEMENT_DATA;
 
   constructor(
     private registrationService: RegistrationService,
-    private formBuilder: FormBuilder,
     private folderService: FolderService,
-    public dialog: MatDialog
-  ) {}
+    private route: ActivatedRoute,
+    private videoService: VideoService,
+    private timeService: TimeService,
+    public dialog: MatDialog,
+    private router: Router ) {
+      route.params.pipe(map(p => p['id']))
+      .subscribe(id=> {
+        this.selectedFolderId = id ;
+      });
+    }
 
   ngOnInit(): void {
     this.getAutorithedUser();
   }
 
+  private getFolders() {
+    return this.folderService.getAllFoldersByUserId(this.currentUser.id).subscribe(
+      (result) => {
+        this.folders = result;
+        var name = this.folders.find(f => f.id == this.selectedFolderId)?.name;
+        if(name === undefined) {
+          this.selectedFolderName='';
+        }
+        else {
+          this.selectedFolderName = ' / '+ name;
+        }
+      }
+    );
+  }
+
+  private getVideos(id: number) {
+    return this.videoService.getAllVideosWithoutFolderByUserId(id).
+        subscribe(res => this.videos = res)
+  }
+
   private getAutorithedUser() {
     return this.registrationService
-      .getUser()
-      .pipe(takeUntil(this.unsubscribe$))
-      .subscribe((user) => (this.currentUser = user));
-  }
-  createFolder(name: string) {
-    let newfolder: NewFolderDto = {
-      name: name,
-      parentId: this.currentFolder,
-      authorId: this.user,
-      teamId: this.team,
-    };
-
-    this.folderService.add(newfolder).subscribe((response) => {
+    .getUser()
+    .pipe(takeUntil(this.unsubscribe$))
+    .subscribe((user) => {
+      if(user.avatarLink === null) {
+        user.avatarLink = '../../assets/icons/test-user-logo.png';
+      }
+      this.currentUser = user;
+      this.getFolders();
+      this.getVideos(user.id);
+    });;
+ }
+
+  createFolder(newFolder: NewFolderDto) {
+    this.folderService.add(newFolder).subscribe((response) => {
       this.folder = response.body as FolderDto;
+      this.getFolders();
     });
   }
 
-  showNewFolderForm() {
-    const dialogRef = this.dialog.open(FolderDialogComponent, {
-      width: '250px',
-      data: { name: '' },
+  updateFolder(folderDto: FolderDto) {
+    this.folderService.updateFolder(folderDto).subscribe(() => {
+      this.getFolders();
     });
+  }
 
-    dialogRef.afterClosed().subscribe((result) => {
-      this.createFolder(result);
-    });
+  showNewFolderForm(folder?: FolderDto) {
+    const dialogConfig = new MatDialogConfig();
+
+    dialogConfig.autoFocus = true;
+    dialogConfig.data = folder === undefined ? '' : folder.name;;
```

Comment:
const dialogConfig : MatDialogConfig = {
autoFocus = true,
data = ...
}

Should be something like this